### ISSUE TITLE
Profit is growing - Balance is less! A bug?

### ISSUE THREAD

**Issue by fm3711** [binary-com/binary-bot#1689]
Now I'm testing a bot on a demo account, and it looks like I have a program bug.
 
During the trading period of 2 days, Friday (July 5) and Monday (July 8) profits were recorded, but there was a loss on the balance sheet.
 
On Friday evening, turn off the bot and take a photo.
Profit is 129.74 - Balance 9848.50
 
Today, 08.07. - turn on the bot.
 
Profit is growing, but then I notice that the balance is less than it was on Friday ?.
Profit 221.37 - And here is the balance already 9787.91
 
How is this possible?? The difference is significant !!
![IMG_20190708_150119](https://user-images.githubusercontent.com/51106582/60817616-ad8e3f80-a1a4-11e9-84ec-b21ab432b7f9.jpg)
![08 07 2019](https://user-images.githubusercontent.com/51106582/60817652-c0a10f80-a1a4-11e9-858e-322a42ab407c.png)

**Comment by kikopqfgdffg** [comment]
send bot?

**Comment by kikopqfgdffg** [comment]
danipq12@gmail.com

**Comment by binary-ex-machina** [comment]
@fm3711 check if you don't have any other bot running or  if you have any API token running on other place

**Comment by fm3711** [comment]
> @fm3711 check if you don't have any other bot running or if you have any API token running on other place

I dont have any other bot for sure, but I have tried the same bot on other account, for half of the day. At that time it gave me loss, and I have turned it off, Maybe it has synchronized somehow? This is not my bot actually, I have bought it, and it is sticked to my account. Maybe this is the reason?

**Comment by hotmatrixx** [comment]
@fm3711
I'm concerned about you buying this bot. Did you recieve it as an XML file that you use on your own account, where you have to load it onto your browser?

I'm concerned that you're somhow linked to someone else's account.

I'm not to sure how that could happen, but I am just... looking out for you.

also, how well tested was the bot before yo bought it?> I don't really know of anyone here that sells their bots directly except for a couple of russian chaps who i think may have been banned? (they've stopped posting here recently).

Please take care when using some else's system. plenty of people come here to sell their wares, yet they are not actually sucessful themselves, at least in my experience.

If you want help putting a bot together yourself, we are happy to help. for free.

**Comment by fm3711** [comment]
Yes, I recieved it as a XML file, and I was asked for my account number before I recieved a bot. And I am testing it on a demo account to be sure that it works good. 
I am also confused why this ballance decrising happened, but after I stopped truing to run a bot on two accounts at the same time, the bot works fine. 
Thank you so much for your responce and for your disire to help! Appreciate it! 
I will continue testing and post my results in some time! 
Thanks!
